src/Propel/Runtime/Collection/OnDemandCollection.php

```diff
@@ -247,7 +256,7 @@ public function exchangeArray($input)
      *
      * @return array
      */
-    public function getArrayCopy()
+    public function getArrayCopy($keyColumn = null, $usePrefix = false)
```

Comment:
This makes `OnDemandCollection::getArrayCopy` compatible with its parent's `ObjectCollection::getArrayCopy`